### PR TITLE
Fabo/1630 sent via voyager

### LINKED ISSUE #1630: Add memo to transactions sent through Voyager
Description:

Add memo to track all txs `Sent via Cosmos Voyager 🚀`

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1630

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/vuex/modules/send.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ export default ({ node }) => {
         account_number: rootState.wallet.accountNumber,
         chain_id: rootState.connection.lastHeader.chain_id,
         gas: String(state.externals.config.default_gas),
-        generate_only: true
+        generate_only: true,
+        memo: `Sent via Cosmos UI 🚀`
```
Comment (fedekunze) [inline app/src/renderer/vuex/modules/send.js]:
```suggestion
        memo: `Sent via Cosmos Voyager 🚀`
```

File: test/unit/specs/store/send.spec.js [hunk 2]
```diff
@@ -135,7 +135,8 @@ describe(`Module: Send`, () => {
               from: `cosmos1demo`,
               gas: `42`,
               generate_only: true,
-              sequence: `0`
+              sequence: `0`,
+              memo: `Sent via Cosmos UI 🚀`
```
Comment (fedekunze) [inline test/unit/specs/store/send.spec.js]:
```suggestion
              memo: `Sent via Cosmos Voyager 🚀`
```

File: test/unit/specs/store/send.spec.js [hunk 3]
```diff
@@ -169,7 +170,8 @@ describe(`Module: Send`, () => {
...
```
Comment (fedekunze) [inline test/unit/specs/store/send.spec.js]:
```suggestion
              memo: `Sent via Cosmos Voyager 🚀`
```

File: test/unit/specs/store/send.spec.js [hunk 4]
```diff
@@ -213,7 +215,8 @@ describe(`Module: Send`, () => {
               from: `cosmos1demo`,
               gas: `500000`,
               generate_only: true,
-              sequence: `0`
+              sequence: `0`,
+              memo: `Sent via Cosmos UI 🚀`
```
Comment (fedekunze) [inline test/unit/specs/store/send.spec.js]:
```suggestion
              memo: `Sent via Cosmos Voyager 🚀`
```

File: app/src/renderer/vuex/modules/send.js (hunk 1 above)
Comment (faboweb) [reply app/src/renderer/vuex/modules/send.js]: Well it is not Voyager anymore in the future

Comment (fedekunze) [reply app/src/renderer/vuex/modules/send.js]: it's still Voyager, until we change the name

Comment (fedekunze) [reply app/src/renderer/vuex/modules/send.js]: (we're not `Cosmos UI` either)

**Pull request merged**